docs/host/pypi.md

```diff
@@ -10,21 +10,90 @@ description: "The Open Web Calendar can be installed as a Python package from Py
 
 ![](https://img.shields.io/pypi/v/open-web-calendar.svg) ![](https://img.shields.io/pypi/pyversions/open-web-calendar.svg)
 
+## Installation
 The Open Web Calendar is available as a Python package on [PyPI]({{link.pypi}}).
 If you have [Python 3](https://www.python.org/) installed, run this to install the package:
 
 ```shell
 pip install open-web-calendar
 ```
+At this point you might get the error message
+```shell
+error: externally-managed-environment
+```
+
+On your productive system, it is strongly recommended to follow the instructions to create a virtual environment using 
+```shell
+python3 -m venv /home/username/venv
+```
+followed by 
+```shell
+/home/username/venv/bin/pip install open-web-calendar
+```
+This will install your open-web-calendar app in the project folder  
+/home/username/venv/lib/python3.12/site-packages/open_web_calendar
 
+## Execution
 After installation, run the Open Web Calendar using [Gunicorn](https://pypi.org/project/gunicorn/):
 
 ```shell
-gunicorn open_web_calendar:app
+/home/username/venv/bin/gunicorn open_web_calendar:app
 ```
 
 You should now see the server running at [http://127.0.0.1:8000](http://127.0.0.1:8000).
 
+However, in case you have installed the WebPage you are going to use for the integration of your individual calendar on the same local machine, it must be accessible to other machines as well.  
+In this case you might want to start the open_web_calendar with the dedicated IP-address of the hosting machine.
+
+```shell
+/home/username/venv/bin/gunicorn -b 192.168.178.7 open_web_calendar:app
+```
+You should now see the server running at [http://192.168.178.7:8000](http://192.168.178.7:8000) not only from your local machine.
```

Comment:
This is still localhost or you might need to find out the IP address....
```suggestion
You should now see the server running at [http://localhost:8000](http://localhost:8000), accessible by everyone in the network.
```

It is a bit hard to tell people how to access it now...

Reply:
This will not work if you use localhost from a third party machine.
E.g. Webserver on 192.168.178.5
open_web_calendar app on 192.168.178.6
WebBrowser on 192.168.178.90

If you use localhost:8000 on the WebBrowser-machine it will go nowhere.

Reply:
Hm... Ok. But 192.168.178.7 is also going nowhere... Maybe, we can add something that one should replace that ip with the IP where the OWC is running. Or maybe, people will figure it out... It is only documentation after all, I am not so bothered about this one being too precise because we cannot know the IP.

Reply:
It is already much better than before - let's leave this for later.